src/sentry/lang/javascript/processor.py

```diff
@@ -837,6 +844,10 @@ def __init__(self, *args, **kwargs):
             "sentry:scrape_javascript", True
         ) is not False and self.project.get_option("sentry:scrape_javascript", True)
 
+        self.use_sourcemap_lru_cache = features.has(
+            "organizations:processing-sourcemap-lru-cache", organization
+        )
```

Comment:
It is probably a bad idea to check a feature flag for every event in event processing, I will check with the team.